src/wiki/putdoc.ts

```diff
@@ -0,0 +1,68 @@
+import { S3 } from 'aws-sdk';
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from 'aws-lambda';
+
+export const handler = async function (
+    event: APIGatewayProxyEvent
+): Promise<APIGatewayProxyResult> {
+    try {
+        const s3 = new S3({
+            accessKeyId: process.env.ACCESS_KEY,
+            secretAccessKey: process.env.SECRET_ACCESS_KEY,
+        });
+        const bucketName = process.env.BUCKET_NAME;
+
+        if (event === null) {
+            throw new Error('event not found');
+        }
+
```

Comment:
there is no create doc in the sql db :/ - pause s3 till we figure out the lambda bug 
everything else related to area api and using the sql db should be functional